### ISSUE TITLE
Fehler 3221225794

### ISSUE THREAD

**Issue by Meandor666** [kaixxx/noScribe#2]
noScribe klingt vielversprechend.

Beim Test mit einem Clip von ein paar Sekunden warf es mir folgenden Fehler aus:

> Fehler im 3. Schritt - Transkription
> Whisper.cpp "main" wurde mit folgendem Fehler beendet: 3221225794

**Comment by kaixxx** [comment]
Danke für die Rückmeldung. Es scheint sich um ein Problem mit älteren Prozessoren zu handeln, die bestimmte beschleunigte Rechenoperationen nicht unterstützen (AVX(2)). 
Hier gibt es eine Version von Whisper.cpp, die auch auf älteren CPUs funktionieren sollte: https://github.com/SubtitleEdit/support-files/blob/master/whisper/whisper.cpp-64-SSE2-2023-01-28.zip (Whisper.cpp ist die Software, in der die KI für die Transkription läuft). Das Problem wird hier genauer beschrieben: https://github.com/SubtitleEdit/subtitleedit/issues/6810
Die Zip-Datei enthält zwei Dateien: "main.exe" und "whisper.dll". Beide müssen in das Programmverzeichnis von noScribe kopiert werden, das sich an folgendem Ort befindet: "C:\Users\<USERNAME>\AppData\Roaming\noScribe\". Ersetzen Sie einfach die dort vorhandenen Versionen dieser Dateien. 
Damit sollte es funktionieren, allerdings seeeeehr langsam, befürchte ich...

**Comment by BabyFnord** [comment]
Ist folgender Fehler auch 
> ein Problem mit älteren Prozessoren (...), die bestimmte beschleunigte Rechenoperationen nicht unterstützen
:
```
Fehler im 3. Schritt - Transkription
Whisper.cpp "main" wurde mit folgendem Fehler beendet: 3221225501
```
Bei mir handelt es sich um einen `Intel(R) Core(TM) i7-3615QM CPU @ 2.30GHz`.

**Comment by kaixxx** [comment]
> Ist folgender Fehler auch

Ja, vermutlich schon. Der Prozessor unterstützt tatsächlich kein AVX2. Probieren Sie mal meinen Vorschlag oben? Oder noch besser: verwenden Sie diese Dateien: https://github.com/SubtitleEdit/support-files/blob/master/whisper/whisper.cpp-32-SSE2-2023-01-28.zip Würde mich interessieren, ob das funktioniert. 
Die nächste Version von noScribe wird automatisch erkennen, ob der Prozessor AVX2 unterstützt, und ansonsten die andere Version von Whisper.cpp laden. Aber vielleicht gibt es hier auch noch ein anderes Problem.

**Comment by BabyFnord** [comment]
> verwenden Sie diese Dateien: https://github.com/SubtitleEdit/support-files/blob/master/whisper/whisper.cpp-32-SSE2-2023-01-28.zip Würde mich interessieren, ob das funktioniert.

Das funktioniert 💯, das Ergebnis finde ich prima (aus Erfahrung mit f4x bewertet). Langsam ist es damit in der Tat, leider. Wäre ein co-processing mit der GPU zusätzlich möglich? Wäre auch eine portable Version möglich, um noScribe als non-admin User auf potenteren Systemen laufen lassen zu können? 

Werden transkodierte Proxy-Audiodateien eigentlich automatisch gelöscht? Wo werden diese abgelegt?

**Comment by kaixxx** [comment]
> Das funktioniert 💯, das Ergebnis finde ich prima (aus Erfahrung mit f4x bewertet). 

Klasse, danke fürs Testen. Ich werde das in die nächste Version integrieren. 

> Langsam ist es damit in der Tat (60' für 1' Audio bei `precise`), leider. Wäre ein co-processing mit der GPU zusätzlich möglich? 

Hier gibt es schon eine Diskussion dazu: https://github.com/kaixxx/noScribe/issues/6

> Wäre auch eine portable Version möglich, um noScribe als non-admin User auf potenteren Systemen laufen lassen zu können?

Wenn ich so darüber nachdenke, gibt es eigentlich keinen zwingenden Grund dafür, dass der noScribe-Installer Admin-Rechte benötigt (außer vielleicht: Starmenüeinträge, Registrierung des Uninstallers). Ich werde das bei der nächsten Version mal ohne probieren. 
Auch jetzt müsste es aber eigentlich schon möglich sein, einfach den gesamten Programmordner von noScribe mit einem USB-Stick auf auf eine anderen Rechner zu transferieren und dort auzuführen (besser nicht direkt vom USB-Stick, wegen der Geschwindigkeit). Der Programmorder findet sich an einem etwas ungewöhnlichen Ort: C:\Users\USERNAME\AppData\Roaming\noScribe

> Werden transkodierte Proxy-Audiodateien eigentlich automatisch gelöscht? Wo werden diese abgelegt?

noScribe erzeugt eine temporäre Audiodatei im TEMP-Ordner, diese wird nach Beendigung des Programms (vermutlich sogar direkt nach Abschluss der Transkription) automatisch gelöscht.

**Comment by kaixxx** [comment]
Ach ja: Der non-admin-installer wurde auch implementiert. Danke für die Idee.
